### PR TITLE
readme: add tools download page

### PR DESCRIPTION
**Pull request by lilin90**
@GregoryIan @queenypingcap PTAL

### CODE REVIEW

**Comment by jinqueeny** [comment]
LGTM. @GregoryIan PTAL

**Comment by queenypingcap** [comment]
@GregoryIan PTAL.

File: download.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+This enterprise tools package includes all the following tools:
+
+- Syncer
+- Loader
+- mydumper
+- binlogctl
+- checker
+- ddl_checker
+- dump_region
+- importer
```
[the remark this answers is not in the document]
Comment (GregoryIan) [reply download.md]: we would remove these things from tarball, just because we don't tell user how to use them

File: download.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- sync_diff_inspector
+
+## TiDB-Binlog
```
Comment (GregoryIan) [inline download.md]: remove ` TiDB-Binlog` and `TiDB-Lightning` tarball downlink?  they are also in `tidb` tarball, let user to get it from `tidb` tarball

File: download.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+| [tidb-binlog-kafka-linux-amd64.tar.gz](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.tar.gz) (the Kafka version) | Linux | amd64 |[tidb-binlog-kafka-linux-amd64.sha256](http://download.pingcap.org/tidb-binlog-kafka-linux-amd64.sha256)|
+
+## TiDB-Lightning
+
+| Package name | OS | Architecture | SHA256 checksum |
+|:---|:---|:---|:---|
+| [tidb-lightning-latest-linux-amd64.tar.gz](https://download.pingcap.org/tidb-lightning-latest-linux-amd64.tar.gz) | Linux | amd64 | [tidb-lightning-latest-linux-amd64.sha256](https://download.pingcap.org/tidb-lightning-latest-linux-amd64.sha256) |
+| [tidb-lightning-v2.1.2-linux-amd64.tar.gz](https://download.pingcap.org/tidb-lightning-v2.1.2-linux-amd64.tar.gz) | Linux | amd64 | [tidb-lightning-v2.1.2-linux-amd64.sha256](https://download.pingcap.org/tidb-lightning-v2.1.2-linux-amd64.sha256) |
+
+## Data Migration
```
Comment (GregoryIan) [inline download.md]: I suggest leave two link 
* https://github.com/pingcap/dm/releases
* (http://download.pingcap.org/dm-latest-linux-amd64.tar.gz

File: download.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lilin90) [reply download.md]: Got it!

File: download.md (hunk 2 above)
Comment (lilin90) [reply download.md]: But the user might do not know about this. A separate link can be more friendly.

Comment (GregoryIan) [reply download.md]: we always want to provide one tarball that contains all binary user need

**Comment by lilin90** [comment]
Since Lightning is also included in the TiDB package, @GregoryIan suggests removing the corresponding link and adding some explanation. I'll update the content later.

**Comment by queenypingcap** [comment]
Good point, thanks @morgo. How about changing the title to " Download the enterprise tools"?

**Comment by morgo** [comment]
> Good point, thanks @morgo. How about changing the title to " Download the enterprise tools"?

The top-level nav already has a lot of options.  How about "Download" and indented under "TiDB Enterprise Tools"?

**Comment by queenypingcap** [comment]
Sounds good.

**Comment by GregoryIan** [comment]
@lilin90 any updates?

**Comment by lilin90** [comment]
@GregoryIan @morgo Updated. PTAL again~ Thanks!

File: tools/download.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+If you want to download the latest version of [Syncer](../tools/syncer.md), [Loader](../tools/loader.md), or [mydumper](../tools/mydumper.md), directly download the tidb-enterprise-tools package, because all these tools are included in this package.
+
+| Package name | OS | Architecture | SHA256 checksum |
+|:---|:---|:---|:---|
+| [tidb-enterprise-tools-latest-linux-amd64.tar.gz](http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz) | Linux | amd64 | [tidb-enterprise-tools-latest-linux-amd64.sha256](http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.sha256) |
+
+This enterprise tools package includes all the following tools:
+
+- Syncer
```
Comment (GregoryIan) [inline tools/download.md]: how about link to their documents

Comment (lilin90) [reply tools/download.md]: Good suggestion. I'll add the link.

Comment (lilin90) [reply tools/download.md]: @GregoryIan I checked just now and I find that I already added the doc link in the above paragraph.

**Comment by lilin90** [comment]
@GregoryIan PTAL again

**Comment by GregoryIan** [comment]
LGTM

**Pull request merged**